pkg/etcdenvvar/etcd_env.go

```diff
@@ -241,6 +241,24 @@ func getHardwareSpeedValues(envVarContext envVarContext) (map[string]string, err
 	return hwspeedhelpers.HardwareSpeedToEnvMap(speed)
 }
 
+func getEtcdDBSize(envVarContext envVarContext) (map[string]string, error) {
+	etcd, err := envVarContext.etcdLister.Get("cluster")
+	if err != nil {
+		return nil, fmt.Errorf("failed to retrieve etcd CR: %v", err)
+	}
+
+	if etcd.Spec.BackendQuotaGiB == 0 {
+		return map[string]string{
+			"ETCD_QUOTA_BACKEND_BYTES": "8589934592",
```

Comment:
Why not use GibibytesToBytesString with 8 so it's a little more clear what the default is

Reply:
+1 .. my bad